kitty: add page (#6676)
tldr-pages/tldr: pages/linux/kitty.md

# kitty

> A fast, feature-rich, GPU based terminal emulator.
> More information: <https://sw.kovidgoyal.net/kitty/>.

> - Open the terminal with a title of `Example`:

Reviewer: ```suggestion
- Open the terminal with a specified title:
```

> - Open a terminal with the specified title for the window:

Reviewer 2: I think we should add a basic example.
```suggestion
- Open a new terminal:

`kitty`

- Open a terminal with the specified title for the window:
```

> - Open the terminal with a title of `Example`:
> 
> `kitty --title {{Example}}`

Reviewer 3: Maybe this could be a bit more generic like in other pages. Probably also worth adding quotes to allow titles with spaces. 👍🏻
```suggestion
- Open a terminal with the specified title for the window:

`kitty --title "{{title}}"`
```

> `kitty --title {{Example}}`

Reviewer: ```suggestion
`kitty --title {{title}}`
```

- Start the theme-chooser builtin:

`kitty +kitten themes`

> - Display `Example.jpg` in the terminal:

Reviewer: ```suggestion
- Display a specified image in the terminal:
```

> `kitty +kitten icat {{Example.jpg}}`

Reviewer: ```suggestion
`kitty +kitten icat {{path/to/image.jpg}}`
```

> `kitty +kitten icat {{path/to/image.jpg}}`

Reviewer 2: Does it have to be a `jpg`? If not it should be
```suggestion
`kitty +kitten icat {{path/to/image}}`
```

> - Put `hooray` into the system clipboard:

Reviewer: ```suggestion
- Copy text from STDIN to the system clipboard
```

> - Put `hooray` into the system clipboard:
> 
> `echo {{hooray}} | kitty +kitten clipboard`

Reviewer 3: ```suggestion
- Copy the contents of stdin to the clipboard:

`echo {{example}} | kitty +kitten clipboard`
```

> `echo {{hooray}} | kitty +kitten clipboard`

Reviewer: ```suggestion
`echo {{text}} | kitty +kitten clipboard`
```
